std_msgs/Header header
string[] detected_classes
float32[] scores
